### PR TITLE
Fix/75 78/admin main page style fix

### PR DESCRIPTION
**Pull request by Nicolai-boiko**

### CODE REVIEW

File: src/app/pages/home-page/home-page.component.scss [hunk 1]
```diff
@@ -1,90 +1,24 @@
...
-  justify-content: center;
-}
-
-.wrapper-drop-box {
-  height: 5%;
-}
-
-.wrapper-description {
-  height: 15%;
+  background: #fafafa;
```
Comment (mhasilau) [inline src/app/pages/home-page/home-page.component.scss:8]: upls use constants colors

**Review by mhasilau — APPROVED** [review]

File: src/app/pages/home-page/home-page.component.scss (hunk 1 above)
Comment (vstrabko) [reply src/app/pages/home-page/home-page.component.scss:8]: Yeah, this color should be in our file for colors.

**Review by vstrabko — CHANGES_REQUESTED** [review]

File: src/assets/locales/en.json [hunk 2]
```diff
@@ -73,8 +73,19 @@
...
+    "other": "other",
+    "others": "others",
+    "perPage": "Items per page",
+    "nextPage": "Next page",
+    "prevPage": "Previous page",
+    "of": "of"
+  },
   "tostr": {
-	"title": "Sucess",
+    "title": "Sucess",
```
Comment (vstrabko) [inline src/assets/locales/en.json:88]: Sorry, I know this is not part of this ticket, but could you also please fix spelling of the word `success`?

**Review by vstrabko — CHANGES_REQUESTED** [review]

File: src/styles.scss [hunk 3]
```diff
@@ -16,14 +16,18 @@ body {
...
+    0 41.8px 33.4px rgba(0, 0, 0, 0.086), 0 100px 80px rgba(0, 0, 0, 0.12);
 }
 
 .page_container {
   padding: 12px 15px 0;
   padding-bottom: $height-footer;
-  background: $background-container;
-  min-height: calc(100vh - 70px);
+  background: #EEF2F7;
+  min-height: calc(100vh - 50px);
```
Comment (mhasilau) [inline src/styles.scss]: why r u replace constants to colors?

File: src/app/pages/home-page/home-page.component.scss (hunk 1 above)
Comment (Nicolai-boiko) [reply src/app/pages/home-page/home-page.component.scss]: done

File: src/assets/locales/en.json (hunk 2 above)
Comment (Nicolai-boiko) [reply src/assets/locales/en.json]: no problem :)

File: src/app/components/header/header.component.html [hunk 4]
```diff
@@ -1,17 +1,33 @@
...
+      'header.createNewSandbox' | translate }}</a>
+  </nav>
+  <!-- <nav class="user-buttons">
+    <a routerLink="/users" routerLinkActive="active" (click)="showSpinner()" mat-raised-button *ngIf="userName"
+      id="candidatesList">{{ 'header.candidatesList' | translate }}</a>
+    <a routerLink="/calendar" routerLinkActive="active" mat-raised-button *ngIf="userName" id="calendar">{{
+      'header.calendar' | translate }}</a>
+    <a routerLink="/sandbox" routerLinkActive="active" mat-raised-button *ngIf="userName" id="createNewSandbox">{{
+      'header.createNewSandbox' | translate }}</a>
+  </nav> -->
```
Comment (vstrabko) [inline src/app/components/header/header.component.html:20]: Why is this one commented out?

**Review by vstrabko — APPROVED** [review]

File: src/app/components/header/header.component.html (hunk 4 above)
Comment (Nicolai-boiko) [reply src/app/components/header/header.component.html:20]: this is for choose one of styles for now

**Pull request merged**